File: src/2015/day1/day1.c
#include <stdio.h>
int main() {
  FILE *file = fopen("2015_day1.txt", "r");
  int floor = 0;
  int ch;
  while ((ch = getc(file)) != EOF) {
    if (ch == '(') {
      floor++;
    } else if (ch == ')') {
      floor--;
    }
  }

  fclose(file);
  printf("Floor reached is %d\n", floor);
  return 0;
}

File: src/2015/day1/day1.js
const path = "./2015_day1.txt"
const file = Bun.file(path)
const text = await file.text();
let floor_n = 0;
for (let i = 0; i < text.length; i++){
	if (text[i] == '(') {
		floor_n++;
	} else if (text[i] == ')') {
		floor_n--;
	}
}
console.log(`Floor no:${floor_n}`);